pytorch_lightning/core/saving.py

```diff
@@ -372,10 +373,20 @@ def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
                     OmegaConf.save(OmegaConf.create(hparams), fp, resolve=True)
                 return
 
-    # saving the standard way
     assert isinstance(hparams, dict)
+    hparams_allowed = {}
+    # drop paramaters which contain some strange datatypes as fsspec
+    for k, v in hparams.items():
+        try:
```

Comment:
Hey @Borda, I was wondering if it makes sense to force dumping to yaml file. It might break reproducibility for some use cases. But for the yaml format, we could also support some conversion like: tensor and numpy array to list, etc...

Reply:
so how else would you test that some value/data are not possible to dum, I have not found any method like `is_dumpable` and most examples use this construct...